data/src/main/kotlin/com/alfresco/content/data/BrowseRepository.kt

```diff
@@ -90,6 +91,25 @@ class BrowseRepository(val session: Session = SessionManager.requireSession) {
         )
     }
 
+    suspend fun createFolder(name: String, description: String, parentId: String?) {
+        val list: HashMap<String, String> = HashMap()
+
+        list["cm:title"] = name
+        list["cm:description"] = description
```

Comment:
Just use the shorthand in kotlin: `mapof("cm:title" to name, "cm:description" to description)`

Reply:
Done